fintorch/layers/explainable/Dropout.py

```
@@ -0,0 +1,11 @@
+import torch.nn as nn
+from fintorch.layers.explainable.ValueCaptureHook import ValueCaptureHook
+
+class Dropout(nn.Dropout):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)